src/components/TweetCreationForm.tsx

```diff
@@ -44,6 +44,7 @@ const StyledPopover = styled(Popover)`
 
 const TweetCreationForm: React.FC = () => {
   const [content, setContent] = useState("");
+  const [show, setShow] = useState(true);
```

Comment:
why did you add "show" if you never use it for visualization?
I did not find any place where you use it at UI

Reply:
Yup it is unused code - it was a try, but with no success. deleted. Alex said that for now it is impossible to get done the second part of the task - to let the modal close after click, so he let it be unsolved till some future changes in architecture of the app.